client/src/components/Funding/Payments.js

```diff
@@ -0,0 +1,193 @@
+import React, { useState } from 'react';
+import Button from '@material-ui/core/Button';
+import { makeStyles } from '@material-ui/core/styles';
+import { CardElement, useStripe, useElements } from '@stripe/react-stripe-js';
+import TextField from '@material-ui/core/TextField';
+import axios from 'axios';
+import {
+  Container,
+  CssBaseline,
+  Grid,
+  Typography,
+  Divider,
+} from '@material-ui/core';
+import { useHistory } from 'react-router-dom';
+
+const useStyles = makeStyles((theme) => ({
+  cardElementWrapper: {
+    height: '40px',
+    display: 'flex',
+    alignItems: 'center',
+    width: 475,
+  },
+  divider: {
+    height: '0.25em',
+    width: '8vh',
+    margin: '3em',
+    backgroundColor: theme.appTheme.background,
+  },
+  paper: {
+    marginTop: theme.spacing(8),
+    display: 'flex',
+    flexDirection: 'column',
+  },
+  grid: {
+    marginRight: '-15px',
+    marginLeft: '-15px',
+    width: 'auto',
+    marginTop: '20px',
+  },
+}));
+
+export default function FundingPayment() {
+  const [isProcessing, setProcessingTo] = useState(false);
+  const [checkoutError, setCheckoutError] = useState();
+  const [price, setPrice] = useState();
+  const [name, setName] = useState('');
+  const classes = useStyles();
+  const history = useHistory();
+  const stripe = useStripe();
+  const elements = useElements();
+
+  const handleCardDetailsChange = (e) => {
+    e.error ? setCheckoutError(e.error.message) : setCheckoutError();
+  };
+  const onSuccessfulCheckout = () => {
+    alert('Success');
+    history.push('/explore');
```

Comment:
Eventually we are also going to need to update the dashboard that the project was funded at this point